tutorials/3d/3d_antialiasing.rst

```diff
@@ -279,3 +284,36 @@ usually unnecessary, but it can provide better visuals on high-end GPUs or for
++--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+| Added blur               | 🟢 None                  | 🟡 Some                  | 🟡 Some                  | 🟡 Some                  | 🟡 Some [#]_             | 🟢 None                  |
+|                          |                          |                          |                          |                          |                          |                          |
++--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+| Ghosting artifacts       | 🟢 None                  | 🔴 Yes                   | 🔴 Yes                   | 🟢 None                  | 🟢 None                  | 🟢 None                  |
++--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+| Performance cost         | 🟡 Medium                | 🟡 Medium                | 🔴 High                  | 🟢 Low                   | 🔴 Very High             | 🟡 Medium                |
++--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+| Forward+                 | ✔️ Yes                   | ✔️ Yes                   | ✔️ Yes                   | ✔️ Yes                   | ✔️ Yes                   | ✔️ Yes                   |
++--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+| Mobile                   | ✔️ Yes                   | ❌ No                    | ❌ No                    | ✔️ Yes                   | ✔️ Yes                   | ✔️ Yes                   |
++--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+| Compatibility            | ✔️ Yes                   | ❌ No                    | ❌ No                    | ❌ No                    | ✔️ Yes                   | ❌ No                    |
++--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+
+
+.. [#] MSAA does not work well with materials with Alpha Scissor (1-bit transparency). This can be mitigated by enabling ``alpha antialiasing`` on the material.
+.. [#] TAA transparency antialiasing is most effective when using Alpha Scissor.
+.. [#] FSR2 transparency antialiasing is most effective when using Alpha Scissor.
```

Comment:
Is there any way to combine these? I think it would be a bit cleaner if they could share a note. It would also help communicate that the limitation is the same in both, so they are equivalent.

Reply:
Yeah, I think the other footnote syntax can do that

Reply:
I have a slight preference for this alternative